packages/react-core/src/components/DataToolbar/DataToolbarChipGroupContent.tsx

```diff
@@ -58,27 +58,25 @@ export class DataToolbarChipGroupContent extends React.Component<DataToolbarChip
       <div
         className={css(
           styles.dataToolbarContent,
-          (numberOfFilters === 0 || isExpanded) && getModifier(styles, 'hidden'),
+          (numberOfFilters === 0 || isExpanded) && styles.modifiers.hidden,
           className
         )}
         {...((numberOfFilters === 0 || isExpanded) && { hidden: true })}
         ref={chipGroupContentRef}
         {...props}
       >
         <DataToolbarGroup
-          className={css(collapseListedFilters && getModifier(styles, 'hidden'))}
+          className={css(collapseListedFilters && styles.modifiers.hidden)}
           {...(collapseListedFilters && { hidden: true })}
           {...(collapseListedFilters && { 'aria-hidden': true })}
         />
         {collapseListedFilters && numberOfFilters > 0 && !isExpanded && (
-          <DataToolbarGroup
-            className={css(getModifier(styles, 'toggle-group-summary'), 'pf-m-filters-applied-message')}
-          >
+          <DataToolbarGroup>
```

Comment:
just making sure the removal of class here was intended

Reply:
Yep, these classes no longer exist in core! @mattnolting can you check that our datatoolbar examples aren't missing classes? https://patternfly-react-pr-3886.surge.sh/documentation/react/components/datatoolbar

Reply:
Confirmed, not in core